ethers-contract/ethers-contract-abigen/src/contract/events.rs

```diff
@@ -220,12 +225,17 @@ impl Context {
     /// Expands into a single method for contracting an event stream.
     fn expand_filter(&self, event: &Event) -> TokenStream {
         let ethers_contract = util::ethers_contract_crate();
+        let alias = self.event_aliases.clone().remove(&event.abi_signature());
+
+        let mut name = util::safe_ident(&format!("{}_filter", event.name.to_snake_case()));
```

Comment:
This should be the else path so that 
`let name = if let Some() ... {} else {};`